### ISSUE TITLE
Double spawn of resources

### ISSUE THREAD

**Issue by quimnuss** [SubnauticaNitrox/Nitrox#236]
![image](https://user-images.githubusercontent.com/4179721/42129071-6b82c422-7cba-11e8-9409-5386c55bfc47.png)

It might be related to fish overspawning. I think everything is spawned two times. It's not related to the number of players. I'll try to find more information debugging.

**Comment by killzoms** [comment]
After a little bit of experimenting the issue with the double spawning that appears to happen with flora appears to be due to
```c#
namespace NitroxClient.GameLogic.Spawning
{
    public class DefaultEntitySpawner : IEntitySpawner
    {
        public Optional<GameObject> Spawn(Entity entity, Optional<GameObject> parent)
```
in NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs

The above method spawns creatures, items, etc. the items that exist before hand on a fresh world are duplicated. Examples of this are flora.

**Comment by quimnuss** [comment]
Should be fixed with #238 

Thanks killzoms!
